river/View.zig

```diff
@@ -421,6 +423,14 @@ pub fn setPendingOutput(view: *Self, output: *Output) void {
     switch (server.config.attach_mode) {
         .top => output.pending.wm_stack.prepend(view),
         .bottom => output.pending.wm_stack.append(view),
+        .aside => {
+            attach_aside(
+                &output.pending.wm_stack,
+                output.pending.tags,
+                &view.pending_wm_stack_link,
+                .pending,
+            );
+        },
```

Comment:
If you make the function take `self: *View` and `output: *Output` as arguments, it would be a bit cleaner IMO.

Reply:
Your suggestion was what I initially did, however there is another place (View.zig:575) where `attach_aside()` is called without an `output` to be passed as a parameter. I don't like the way it looks either so If you have any better ideas I'd be glad to see.